app/styles/nav-item.less

```diff
@@ -0,0 +1,33 @@
+.oss-navbar-item {
+  .border-radius-md;
+  padding: var(--spacing-px-6) var(--spacing-px-12);
+  display: flex;
+  flex-direction: row;
+  gap: var(--spacing-px-6);
```

Comment:
from the design, I see a gap of 12px between elements

Reply:
Did you try stacking multiple in a column ?
I have the hunch that the icons are not aligned.

Define a fixed witdth div around the icon & center icons in this div

Reply:
Fixed issue of gap & icon wrapper
<img width="524" alt="Capture d’écran 2024-04-04 à 10 17 14" src="https://github.com/upfluence/oss-components/assets/37589993/459ed6ca-79f4-4eaf-9e8a-bda891d6612e">